packages/ra-ui-materialui/src/detail/Edit.spec.tsx

```diff
@@ -831,7 +831,7 @@ describe('<Edit />', () => {
                     </Edit>
                 </AdminContext>
             );
-            await screen.findByText('Foo #123');
+            await screen.findByText('Foo lorem');
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** The test expect `defaultTitle` to use the record id by default. But that is no longer the case.. We need to change the test title or remove the `title` in the `getOne` result.
What do you think @djhi ?‏

Reply:
Indeed